vtm/src/org/oscim/theme/XmlThemeBuilder.java

```diff
@@ -1299,9 +1299,9 @@ private void tagTransform(String localName, Attributes attributes) {
         }
 
         if (v == null && matchValue == null) {
-            mTransformKeyMap.put(matchKey, k);
+            mTransformKeyMap.put(k, matchKey);
         } else {
-            mTransformTagMap.put(new Tag(matchKey, matchValue), new Tag(k, v));
+            mTransformTagMap.put(new Tag(k, v), new Tag(matchKey, matchValue));
```

Comment:
Why the reverse (please explain the changes in detail) ?

Reply:
As said, rule is:
`k` is transformed to `matchKey`
`v` is transformed to `matchValue`

One `k` can only have one `matchKey`, as they're all processed in the same rule.
But one  `matchKey` can have multiple `k`.
See example in first post with colors, to understand. So the map is wrong directed to represent this relation.

Same for Tags with values.